internal/cmd/integration-tests/tests/scrape-prom-metrics/config.alloy

```diff
@@ -1,28 +1,29 @@
+logging {
+    level = "debug"
+}
+
+livedebugging {}
+
 prometheus.scrape "scrape_prom_metrics" {
   targets = [
     {"__address__" = "prom-gen:9001"},
   ]
-  forward_to = [prometheus.remote_write.scrape_prom_metrics.receiver, otelcol.receiver.prometheus.scrape_prom_metrics_to_otlp.receiver]
+  forward_to = [prometheus.write.queue.scrape_prom_metrics.receiver, otelcol.receiver.prometheus.scrape_prom_metrics_to_otlp.receiver]
   scrape_classic_histograms = true
-  scrape_protocols = ["PrometheusProto", "OpenMetricsText1.0.0", "OpenMetricsText0.0.1", "PrometheusText0.0.4"]
+  scrape_native_histograms = true
   scrape_interval = "1s"
   scrape_timeout = "500ms"
 }
 
-prometheus.remote_write "scrape_prom_metrics" {
-  endpoint {
-    url = "http://mimir:9009/api/v1/push"
-    send_native_histograms = true
-    metadata_config {
-        send_interval = "1s"
+prometheus.write.queue "scrape_prom_metrics" {
```

Comment:
As a follow up, I want to expand our integration tests to cover more cases and protocols and use old remote_write component too. But for now we should switch to the one we use most commonly: new WAL with (mostly) default settings. The only settings we're giving here are to make the test go faster.